packages/mds-geography-author/validators.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2021 City of Los Angeles
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SchemaValidator } from '@mds-core/mds-schema-validators'
+import { Geography } from '@mds-core/mds-types'
+
+const uuidSchema = { type: 'string', format: 'uuid' }
+const timestampSchema = { type: 'integer', minimum: 100_000_000_000, maximum: 99_999_999_999_999 }
+const stringSchema = { type: 'string', transform: ['trim'], minLength: 1 }
+
+export const { validate: validateGeographyDetails } = SchemaValidator<Geography>({
```

Comment:
There's _gotta_ be a GeoJSON AJV validator that already exists somewhere in npm, it may be better to use that.

Reply:
Aha, there is! I'm a little confused that this exists when `validateGeographyDomainCreateModel` does too, they seem to be talking about the same thing...

Reply:
yes! But, given the massive scope of this PR, i didn't also want to deprecate all the `mds-db` usage at the same time as well.